packages/mdc-ripple/foundation.js

```diff
@@ -146,12 +148,20 @@ export default class MDCRippleFoundation extends MDCFoundation {
 
     activationState.isActivated = true;
     activationState.isProgrammatic = e === null;
+    activationState.isTimeoutDeactivated = false;
```

Comment:
I think this, as well as `isProgrammatic`, need to be added to the return value of `defaultActivationState_()`